lib/reporting/identity_verification_report.rb

```diff
@@ -77,6 +77,14 @@ def progress?
       @progress
     end
 
+    def identity_verification_emailable_report
+      EmailableReport.new(
+        title: 'Identiy Verification Metrics',
+        table: to_csv,
```

Comment:
`table` should be an array of arrays, not a single string, I would break `to_csv` into two methods:
1. `as_csv` , build the arrays of strings (rows)
2. `to_csv`, takes `as_csv` and makes a single string to preserve existing behavior